### DISCUSSION TITLE
Does it work?

### DISCUSSION THREAD

**Discussion by muhkuh666** [nathanmarlor/foxess_modbus#477] [Q&A] [answered]
So as Foxess changed API and cloud solution will not work anymore with HA, I need a different solution. I have H3-6.0-E inverter located in my basement! So I need a solution where I would then like to access the data from the Modbus interface via Ethernet and process it with my Home Assistant. Is that possible? Which device can I use to send the MODBUS data from the inverter to the local Ethernet where my HA Server is in?

**Comment by canton7** [comment, marked as answer]
Yes, that's what this integration does. See the links in the sidebar of the wiki [here](https://github.com/nathanmarlor/foxess_modbus/wiki) for modbus to ethernet adapters.
